released/packages/coq-mathcomp-ssreflect/coq-mathcomp-ssreflect.2.4.0/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+maintainer: "Mathematical Components <mathcomp-dev@sympa.inria.fr>"
+
+homepage: "https://math-comp.github.io/"
+bug-reports: "https://github.com/math-comp/math-comp/issues"
+dev-repo: "git+https://github.com/math-comp/math-comp.git"
+license: "CECILL-B"
+
+depends: [
+  "coq-core"
```

Comment:
Why the dependency?

Reply:
This looks indeed spurious to me, since it is transitively satisfied by depending on `rocq-mathcomp-ssreflect.2.4.0`. I'll remove.

Reply:
Otherwise we could `opam install coq-mathcomp-ssreflect` and end up without Coq, which I'd personally find very disturbing. If users don't want coq, they'll install rocq-mathcomp-ssreflect. Note that we already have a similar constraint in all those metapackages, for instance `coq-stdlib` or `coq-elpi`.
(That being said, it seems to be missing in the `coq-equations.1.3.1+9.0` metapackage Cc @mattam82 (btw, this package also has an explicit `=1.3.1+9.0` whereas `=version` would probably be easier to use))

Reply:
Ok, thanks for the explanation!

Reply:
Ok, I can see you fixed it. Another option is to make all the `coq-mathcomp-*` packages depend on `coq-mathcomp-ssreflect`, to match the dependency structure on the `rocq-mathcomp-*` side. The advantage of this approach would be that the dependency structure is kept in the compatibility layer, but I'm not sure whether it has practical implications either way.

Reply:
That's strictly equivalent but longer to type and less principled, so probably not worth it.

Reply:
Not quite: say you depend on `coq-mathcomp-algebra`, then you would avoid installing `coq-mathcomp-ssreflect` in the current state. Functionally, it's the same, but you end up with a different set of packages, which might be confusing.